packages/homepage/content/articles/new-dashboard/index.md

```diff
@@ -3,34 +3,33 @@ banner: ./images/banner.png
 slug: new-dashboard
 authors: ['Ives van Hoorne']
 photo: https://avatars0.githubusercontent.com/u/587016?s=460&v=4
-title: Introducing Workspaces
+title: Introducing Teams
```

Comment:
A bit tricky, but wouldn't it be lead to misunderstanding the difference between Team Pro and Teams?

Reply:
Yes, there's a chance that there could be some confusion here because of the rename, but I still think it's ok (and discussed it lots with @DannyRuchtie to decide what to do here)